Mathlib/RingTheory/HahnSeries/Addition.lean

```diff
@@ -150,7 +150,7 @@ theorem min_orderTop_le_orderTop_add {Γ} [LinearOrder Γ] {x y : HahnSeries Γ
     WithTop.coe_le_coe]
   exact HahnSeries.min_le_min_add hx hy hxy
 
-theorem min_order_le_order_add {Γ} [Zero Γ] [LinearOrder Γ] {x y : HahnSeries Γ R}
+theorem min_order_min_order_le_add {Γ} [Zero Γ] [LinearOrder Γ] {x y : HahnSeries Γ R}
```

Comment:
Should this be called `min_order_le_add`?

Reply:
It looks like it is a typo, it was docs#HahnSeries.min_order_le_order_add

Reply:
OK, I will restore it, thanks!